Date: Jan 15, 2016 2:25 AM

From: 74df33a08b6ccc08704fecc6c4874195d5888f06 Jan 15, 2016 2:32 AM

Subject: [PATCH] ARRISXB3-4070 : Log files created is not becoming 0 bytes

        Some log4c patches were missing in Yocto

Source: COMCAST

License: LGPL-2.0-or-later

Upstream-Status: Pending

Signed-off-by: achelakkode <anoopnair_c@infosys.com>
---

diff --git a/src/log4c/rollingpolicy_type_sizewin.c b/src/log4c/rollingpolicy_type_sizewin.c
index cec35bd..dc16dd2 100644
--- a/src/log4c/rollingpolicy_type_sizewin.c
+++ b/src/log4c/rollingpolicy_type_sizewin.c
@@ -389,8 +389,10 @@ static int sizewin_get_last_index(rollingpolicy_sizewin_udata_t *swup){
 static int sizewin_open_zero_file(char *filename, FILE **fpp ){
   int rc = 0;
   sd_debug("sizewin_open_zero_file['%s'", filename);
+//printf(">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>sizewin_open_zero_file['%s'", filename);
 
-  if ( (*fpp = fopen(filename, "w+")) == NULL){
+  //if ( (*fpp = fopen(filename, "w+")) == NULL){
+   if ( (*fpp = fopen(filename, "a+")) == NULL){
    sd_error("failed to open zero file '%s'--defaulting to stderr--error='%s'",
      filename, strerror(errno));    
    *fpp = stderr;

